doc/specs/vulkan/chapters/devsandqueues.txt

```diff
@@ -347,6 +347,13 @@ include::../structs/VkDeviceCreateInfo.txt[]
 
 include::../validity/structs/VkDeviceCreateInfo.txt[]
 
+fname:vkCreateDevice creates a VkDevice object, then enables
+extensions requested by the application. If an extension
+is provided by a layer, the layer must: be specified at
```

Comment:
Here's definition of "must":
This word, or the terms **required** or **shall**, mean that the definition is an absolute requirement of the specification.

It's still required that the layer must be specified at CreateInstance time to get the desired behavior. The application **must** explicitly enable functionality it wants to use.